README.md

```diff
@@ -1617,6 +1621,7 @@ Update Time, five active automations, webhooks.
    * [wakatime.com](https://wakatime.com/) — Quantified self-metrics about your coding activity using text editor plugins, limited plan for free.
    * [WebComponents.dev](https://webcomponents.dev/) — In-browser IDE to code web components in isolation with 58 templates available, supporting stories, and tests.
    * [PHPSandbox](https://phpsandbox.io/) — Online development environment for PHP.
+   * [Xcode For MacOS](https://developer.apple.com/xcode/) -Xcode For MacOS
```

Comment:
also pretty much just part of the OS and generally not an option to not use in reality, so i'd rather not add a redundant entry to an already massive list